### ISSUE TITLE
Exception when using JpaStorageProvider

### ISSUE THREAD

**Issue by Fiurgeist** [electronicarts/orbit#51]
Hi,
I switched my little test project to MySQL and get now the following exception
```
com.ea.orbit.exception.UncheckedException: java.lang.IllegalArgumentException: An exception occurred while creating a query in EntityManager:
Exception Description: Problem compiling [select s from State s where s.stateId=:stateId].
```
coming from
```
c.ea.orbit.actors.runtime.Execution - Error reading actor state for: com.ea.orbit.actors.runtime.ReminderControllerFactory$ReminderControllerReference
```
because the ReminderController has a state as inner class (and no annotation), and that is not supported by the implementation of JpaStorageProvider according to your documentation.
Not sure if there are other Orbit internal actors with state, but that's the one I found.

**Comment by quilombodigital** [comment]
Hi, 

Since I did the initial implementation of the JPAProvider, I´ll take a look at this at night.

Thanks

**Comment by JoeHegarty** [comment]
Thanks guys. Let us know if you need us to take a look.
It's a holiday in Canada today, but we'll be back in the office tomorrow.

**Comment by JoeHegarty** [comment]
This should be fixed in PR #52 thanks to @quilombodigital.
It will be included in the next official release.

**Comment by quilombodigital** [comment]
Made the PR. :)

**Comment by quilombodigital** [comment]
Heh.. Your faster than me Joe. :)

**Comment by Fiurgeist** [comment]
nice^^

**Comment by JoeHegarty** [comment]
This is resolved in the 0.3.0 release.
https://github.com/electronicarts/orbit/releases/tag/v0.3.0
